tests/CMakeLists.txt

```diff
@@ -104,6 +113,16 @@ if((PYBIND11_TEST_FILES_ASYNC_I GREATER -1) AND (PYTHON_VERSION VERSION_LESS 3.5
   list(REMOVE_AT PYBIND11_TEST_FILES ${PYBIND11_TEST_FILES_ASYNC_I})
 endif()
 
+# Skip tests for CUDA check:
+# /pybind11/tests/test_constants_and_functions.cpp(125):
+#   error: incompatible exception specifications
```

Comment:
What does "incompatible exception specification" mean?

Reply:
That's printed by NVCC when running. It doesn't like `noexcept(false)` or `throw()`, I believe, but the error message line is off (I think), so it's hard to tell what it is unhappy with.